Assets/CodeBase/Localization/LocalizationKeyData.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+
+[CreateAssetMenu(fileName = "LanguageData", menuName = "Localization/Language Data")]
+public class LocalizationKeyData : ScriptableObject
+{
+    public List<LocalizationKey> Translations;
+}
+
+[Serializable]
```

Comment:
Dictionaries cannot be serialized so this is a small work around. I'll look into a better solution in the future, but this will work for the time being.